css/dubplus.css

```diff
@@ -384,22 +421,24 @@
       background-color: rgba(255, 255, 255, 0.3); }
 
 .emoji-picker-image {
-  background-image: url(https://cdn.jsdelivr.net/gh/DubPlus/DubPlus@beta/images/emoji-spritesheet.png); }
+  background-image: url(https://cdn.jsdelivr.net/gh//DubPlus@beta_fixes/DubPlus_Logo_Owners/images/emoji-spritesheet.png); }
```

Comment:
you need to have a local copy of `beta` branch and run the build then so that this just says `DubPlus@beta` and not `beta_fixes/DubPlus_Logo_Owners`.  That or once this is merged, pull down beta locally build and update again.  

However!

I think I want to get away from having the images accessed from the repos.  Either want to move them to imgur OR our main site like you did for the logos in cat:

https://dub.plus/assets/img/logo.svg

Reply:
I'll switch the image URLs to use https://dub.plus/assets/img/logo.svg, and I'll merge this to beta and rebuild

Reply:
Updated the logo URLs as well as beta links